pkg/netpol/eval/internal/k8s/service.go

```diff
@@ -38,6 +39,10 @@ func ServiceFromCoreObject(svcObj *corev1.Service) (*Service, error) {
 		return nil, errors.New("K8s Service without selectors is not supported")
 	}
 
+	if svc.Namespace == "" {
+		svc.Namespace = metav1.NamespaceDefault
+	}
+
```

Comment:
doesn't `scan` pkg assigns `default` namespace for resources without a namespace?

Reply:
True